addon/-private/observer-admin.js

```diff
@@ -15,20 +15,26 @@ import IntersectionObserverAdmin from 'intersection-observer-admin';
 export default class ObserverAdmin {
   /** @private **/
   constructor() {
-    this.ioAdmin = new IntersectionObserverAdmin();
+    this.instance = new IntersectionObserverAdmin();
   }
 
   /**
    * @method add
    * @param HTMLElement element
    * @param Object observerOptions
-   * @param HTMLElement|window scrollableArea
    * @param Function enterCallback
    * @param Function exitCallback
    * @void
    */
-  add(...args) {
-    return this.ioAdmin.observe(...args);
+  add(element, observerOptions, enterCallback, exitCallback) {
+    if (enterCallback) {
+      this.instance.addEnterCallback(element, enterCallback);
```

Comment:
Here we could add an API to `addEnterCallback` with an element && function.  The user would need to so something like `inViewport.addEnterCallback(element, callback)`. When the element intersects or exists, we (internal library) dispatches an event to call that callback.  This would allow us to be tied only as a mixin.